tools/devops/automation/build-pipeline.yml

```
@@ -93,8 +93,8 @@ parameters:
     {
       stageName: 'mac_10_13',
       displayName: 'Mac High Sierra (10.13)',
-      macPool: 'Hosted Mac Internal',
-      useImage: false,
+      macPool: 'internal-macos-10.1',
```

```suggestion
      macPool: 'internal-macos-10.13',
```
Should this one be 10.13?